action.yml

```diff
@@ -56,8 +59,15 @@ runs:
           # Run the script that creates a new release and deploys it
           solsta_extra_args=""
           if [ ! "${{ inputs.release_version }}" = "" ]; then
-            solsta_extra_args="--version=${{ inputs.release_version }}"
+            solsta_extra_args+="--version=${{ inputs.release_version }}  "
+          fi
+          if [ ! "${{ inputs.hidden_files }}" = "" ]; then
+            VAL=${{ inputs.hidden_files }}
+            for i in ${VAL[@]} ; do
+              solsta_extra_args+=" --hidden_file=$i "
+            done
           fi
+          echo $solsta_extra_args
```

Comment:
I think some debug code was left in here.

Reply:
Argh, you're right!